src/dvclive/live.py

```diff
@@ -191,6 +198,78 @@ def log_plot(self, name, labels, predictions, **kwargs):
         data.dump(val, self._step, **kwargs)
         logger.debug(f"Logged {name}")
 
+    def log_params(
+        self, params: Dict[str, ParamsLike], name: str = PARAMS_DEFAULT_NAME
+    ):
+        """Saves the given set of parameters (dict) to json"""
+        if self._resume:
+            logger.debug("resume=True, not removing existing parameters.")
+            return
+
+        output_dir = Path(self.dir) / self.PARAMS_DIR
+        output_dir.mkdir(exist_ok=True)
+        params_path = output_dir / (
+            f"{name}.json" if not name.endswith(".json") else name
+        )
```

Comment:
(Implementation)
Not a blocker but, for consistency, I think would be nice if we implement a new `Data` subclass (i.e. `Param`) and move this logic there.

It would look more similar to the other `log_x` methods and could launch explicit exceptions for invalid data types, etc.

Reply:
I thought about that, and that's the first approach I went for, although if you look at the `Data` class most of its (abstract) methods are tightly connected to the "stepped" workflow:  `step`, `no_step_output_path`, `first_step_dump`, `no_step_dump`, `step_dump`. Maybe have a new superclass from which `Data` and `Param` derive, although I think they would share only few methods/attributes.

I think a `Param` class is something we will have to add, especially in relation to the nested dictionaries/complex objects parameters support.